packages/opencv/lib/index.js

```diff
@@ -344,57 +294,42 @@ async function getImagesMatches(img1Data, img2Data, options = {}) {
       img1.delete();
       img2.delete();
       detector.delete();
-      result1.keyPoints.delete();
-      result1.descriptor.delete();
-      result2.keyPoints.delete();
-      result2.descriptor.delete();
+      result1?.keyPoints.delete();
+      result1?.descriptor.delete();
+      result2?.keyPoints.delete();
```

Comment:
I assume the `delete()` would still fail in this case if result is null

Reply:
if `result1` is `null`, the method will not be called.

Reply:
I'm still not quite sure about that. Just tried 

```
let users = {
  'u1': { foo: () => 'yolo' },
};

result = users?.u4.foo()
```
in the console and got `users.u4 is undefined` error

Reply:
I think @boneskull is saying that if `result1` exists, we can assume that it will have a `keyPoints` object with a `delete` method on it, because that's guaranteed by the nature of what `result1` is